fix: update copy for when we get 202 from ClickHouse

fuziontech opened the pull request:
just a tweak on the message we give customers when things are hot

fuziontech committed:
fix: update copy for when we get 202 from ClickHouse

raquelmsmith commented on posthog/exceptions.py:
```diff
@@ -45,7 +45,7 @@ class Conflict(APIException):
 class ClickhouseAtCapacity(APIException):
     status_code = 500
-    default_detail = "Clickhouse cluster is at capacity. Please try this query again later."
+    default_detail = "We are a little busy right now. Please try this query again later."
```
```suggestion
    default_detail = "Queries are a little too busy right now. We're working to free up resources. Please try again later."
```

Maybe? So it doesn't feel like "posthog gets busy and then just dies"

fuziontech committed:
Update posthog/exceptions.py

Co-authored-by: Raquel Smith <raquelmsmith@users.noreply.github.com>

fuziontech committed:
Merge branch 'master' into fix_copy_202

fuziontech committed:
fix: Update ClickhouseAtCapacity test to match actual error message

The test was expecting an old error message that no longer matches
the actual default_detail in the ClickhouseAtCapacity exception class.
Updated the test to expect the correct message:
'Queries are a little too busy right now. We're working to free up resources. Please try again later.'

fuziontech committed:
fixes
